src/appendix/2-how-to-use-gdb/README.md

```diff
@@ -0,0 +1,87 @@
+# How to use GDB
+
+Below are some useful GDB commands that can help us debug our programs. This assumes you have [flashed a program](../../05-led-roulette/flash-it.md) onto your microcontroller and attached to an OpenOCD session.
+
+## General Debugging
+
+> **NOTE:** Many of the commands you see below can be executed using a short form. For example, `continue` can simply be used as `c`, or `break $location` can be used as `b $location`. While you are learning GDB try to see how short you can get the commands to go before GDB doesn't recognize it!
+
+
+### Dealing with Breakpoints
+
+* `break $location`: Set a breakpoint at a place in your code. The value of `$location` can include:
+    * `b 123` - Break on line 123 of the currently displayed file
+    * `b main.rs:123` - Break on line 123 of `main.rs`
+* `info break`: Display current breakpoints
+* `delete`: Delete all breakpoints
+    * `d $n`: Delete breakpoint `$n`
+* `clear`: Delete breakpoint at next instruction
+    * `clear main.rs:$function`: Delete breakpoint at entry of `$function` in `main.rs`
+    * `clear main.rs:123`: Delete breakpoint on line 123 of `main.rs`
+* `enable`: Enable all set breakpoints
+  * `en $n`: Enable breakpoint `$n`
+* `disable`: Disable all set breakpoints
+  * `dis $n`: Disable breakpoint `$n`
+
+### Controlling Execution
+
+* `continue`: Begin execution of your program
```

Comment:
I'd prefer something like "Begin or continue execution of your program" but I guess that's obvious from the name...

Reply:
Better to be explicit, I added your suggestion. Nice catch.